src/main/java/frc/robot/config/Config.java

```diff
@@ -125,10 +125,12 @@ private Config() {
     public static NetworkTable constantsTable = NetworkTableInstance.getDefault().getTable("constants");
 
     // Vision Table Constants
-    public static String VISION_TABLE_NAME = "MergeVision";
-    public static String DISTANCE_POWERCELL = "DistanceToPowerCell";
-    public static String YAW_POWERCELL = "YawToPowerCell";
-    public static String YAW_OUTER_PORT = "YawToTarget";
+    public static String VISION_TABLE_NAME      = "MergeVision";
+    public static String DISTANCE_POWERCELL     = "DistanceToPowerCell";
+    public static String DISTANCE_OUTER_PORT    = "DistanceToOuterPort";
+    public static String YAW_POWERCELL          = "YawToPowerCell";
+    public static String YAW_OUTER_PORT         = "YawToTarget";
+    //public static String ANGLE_OUTER_PORT       = "AngleToOuterPort";
```

Comment:
Do we need this constant?

Reply:
We probably need this later, as this is the third angle given to us by vision. This angle can be used to decide whether we can target the inner port or the outer port. Or I can delete this for now.